### PR TITLE
Updated quota limits as per bug:1458852

### PR DESCRIPTION
**Pull request by punituee**
Changed default quota of Nova Instances.

Except Blocks Volumes per Instance: 20. Hoping to take this up on another commit.

closes bug: 1458852

### CODE REVIEW

**Comment by punituee** [comment]
please test this

**Comment by punituee** [comment]
please test this

**Comment by punituee** [comment]
The functionality has been tested on jenkins buid #1309.
$ nova quota-show
+-----------------------------+-------+
| Quota                       | Limit |
+-----------------------------+-------+
| instances                   | 20    |
| cores                       | -1    |
| ram                         | -1    |
| floating_ips                | 10    |
| fixed_ips                   | -1    |
| metadata_items              | 128   |
| injected_files              | 5     |
| injected_file_content_bytes | 10240 |
| injected_file_path_bytes    | 255   |
| key_pairs                   | 10    |
| security_groups             | 10    |
| security_group_rules        | 20    |
+-----------------------------+-------+

Build failed due to public cloud outage. 
Recommend merging.

**Comment by sorenh** [comment]
If the tests failed, we don't know if this will break anything. Running tests again.

**Comment by sorenh** [comment]
please test this

File: hiera/data/common.yaml [hunk 1]
```diff
@@ -341,10 +341,10 @@ nova::conductor::enabled: true
...
 nova::cert::enabled: true
 
-nova::quota::quota_instances: 50
-nova::quota::quota_cores: 100
-nova::quota::quota_ram: 51200
-nova::quota::quota_volumes: 10
+nova::quota::quota_instances: 20
+nova::quota::quota_cores: -1
+nova::quota::quota_ram: -1
+nova::quota::quota_volumes: 120
```
Comment (bodepd) [inline hiera/data/common.yaml]: this setting does not seem to be used. I don't see it when I run nova quota-show.

It also looks like it is not being set in cinder, when I run:

````
# just running for testtenant on a local build
cinder quota-show 801eb520ce434b78bf51cc21049a79e4

+-----------+-------+
|  Property | Value |
+-----------+-------+
| gigabytes |  1000 |
| snapshots |   10  |
|  volumes  |   10  |
+-----------+-------+
````

**Comment by bodepd** [comment]
One note about potential process. It would be great to include the full link (so that a reveiwer can just click on it) to related launchpad tickets as a separate line, perhaps as the first line in the commit message?

File: hiera/data/common.yaml (hunk 1 above)
Comment (punituee) [inline hiera/data/common.yaml]: yes. as It turns out quota_volumes is Deprecated.
cinder quota-update (tenant-id) ----volumes=120 will fix this. 
ref: https://bugs.launchpad.net/horizon/+bug/1095876

**Comment by punituee** [comment]
Agreed! I will make sure of that next time.

Updating/adding the quota_volumes=1000 in cinder.conf is not currently supported via puppet script it seems. Workaround is updating the limits with admin on cloud till we have a solution.

**Comment by bodepd** [comment]
why would setting quota_volumes not be supported by Puppet? It most certainly is.

**Comment by bodepd** [comment]
and just to clarify, any workarounds that require manual intervention will never be acceptable. We need a completely repeatable installation process that is as consistent as possible between all environments.

Even if the answer is to run a script, that script needs to be run from Puppet and included as a part of this repo.

**Comment by punituee** [comment]
My Bad. I am creating a new pull request waking into account the cinder volume quota limit too and closing this.

**Comment by bodepd** [comment]
it's also possible to just repush to the same branch and it'll update the PR. Opening a new one will loose the context of the conversation so far.

**Comment by bodepd** [comment]
and not worries about not fully understanding the entire process. It's all meant to be a learning exercise.

**Comment by punituee** [comment]
yeah Dan and Soren thanks for all the help here. I have repushed.

**Comment by punituee** [comment]
please test this

**Comment by punituee** [comment]
please test this

**Comment by punituee** [comment]
all green. quotas reflecting. Recommend merging.

**Pull request merged**